core/src/main/java/org/apache/brooklyn/util/core/osgi/BundleMaker.java

```diff
@@ -110,8 +110,6 @@ public File createJarFromClasspathDir(String path) {
                 addUrlItemRecursively(zout, path, path, Predicates.alwaysTrue());
             }
             
-            Streams.closeQuietly(zout);
```

Comment:
Should this be removed?

Reply:
Yes - it's done in the finally block as well, so this line is redundant (admittedly not to do with deprecated code, but is general code cleanup).